src/lib/web-component/log-view/log-view-element.js

```diff
@@ -63,19 +136,28 @@ export class LogView extends HTMLElement {
     this.logTableWrapper.classList.remove('width-60');
   }
 
-  handleEvent(event) {
-    if (event.type === 'click') {
-      const logDetails = event.target.closest('tr')?._log;
+  hideTSFilter() {
+    this.startTimestamp = null;
+    this.stopTimestamp = null;
+    this.disptachTSFilterEvent();
 
-      if (logDetails) {
-        this.openDetailSidebar(logDetails);
-        return;
-      }
+    this.startTimeLabel.textContent = 'From Beginning';
+    this.stopTimeLabel.textContent = 'Up to End';
+    this.timeStampFilterContainer.hidden = true;
+  }
 
-      if (event.target === this.closeBtn) {
-        this.closeDetailSidebar();
-      }
-    }
+  createContextMenu() {
+    browser.menus.create({
```

Comment:
The current parameters create a menu that is visible in all contexts, on all pages. This is very undesirable.

You should only create a menu that matches the current context. For example, see https://piro.sakura.ne.jp/latest/blosxom/mozilla/xul/2018-10-14_override-context-on-fx64.htm#_adding_menu_items_with_the_viewtypes_parameter_

(I recommend to read that whole blog post if you need more context on why the API exists and how to use it)